### ISSUE TITLE
failed to build webui for 2.14.0 release

### ISSUE THREAD

**Issue by laoshaw** [openbmc/openbmc#3929]
**Note on Bugs**
Please note that questions or feature requests should be discussed on the
mailing list or Discord. Please see https://github.com/openbmc/openbmc#contact
for info on how to contact us. Github Issues in openbmc/openbmc are for bugs
only. Please remove this section when opening a bug.

**Unexpected behavior you saw**

**Expected behavior**
failed to build webui for 2.14.0 release

**To Reproduce**

. setup romulus
time bitbake obmc-phosphor-image

error message:

```
| npm http fetch GET 200 https://registry.npmjs.org/@vue%2fcli-plugin-eslint 116ms (cache miss)
| npm http fetch GET 200 https://registry.npmjs.org/eslint 29ms (cache hit)
| npm ERR! code ERESOLVE
| npm ERR! ERESOLVE unable to resolve dependency tree
| npm ERR!
| npm ERR! While resolving: webui-vue@0.1.0
| npm ERR! Found: eslint@7.22.0
| npm ERR! node_modules/eslint
| npm ERR!   dev eslint@"7.22.0" from the root project
| npm ERR!
| npm ERR! Could not resolve dependency:
| npm ERR! peer eslint@">= 1.6.0 < 7.0.0" from @vue/cli-plugin-eslint@4.5.12
| npm ERR! node_modules/@vue/cli-plugin-eslint
| npm ERR!   dev @vue/cli-plugin-eslint@"4.5.12" from the root project
| npm ERR!
| npm ERR! Fix the upstream dependency conflict, or retry
| npm ERR! this command with --force or --legacy-peer-deps
| npm ERR! to accept an incorrect (and potentially broken) dependency resolution.
```

**Screenshots**

**OpenBMC Information:**
 - /etc/os-release: ubuntu 22.04
 - uname -a:  - Any additional patches or changes: X550P7C 5.19.0-41-generic #42~22.04.1-Ubuntu SMP PREEMPT_DYNAMIC Tue Apr 18 17:40:00 UTC 2 x86_64 x86_64 x86_64 GNU/Linux

**Additional context**

**Comment by williamspatrick** [comment]
I just built romulus from the 2.14.0 tag to confirm.  It worked fine.  Have you made any changes you've not documented here?

The error posted is with NPM, which is well outside our control.  This seems like a network issue?

**Comment by williamspatrick** [comment]
The recipes will pick up whatever is in the packages-lock file in the webui repository. They’ve not fully Yocto-fied the recipe yet.

**Comment by bassdanny** [comment]
I was having this issue as well. I'm half curious if there isn't an NPM version dependency here possibly? I'm new and have yet to get the project to build and work properly as of yet, so there must be some other needed setup/packages that I need to install first. Just a guess... I'll let you know if I happen to figure it out. I don't think it's a problem with the project, more so just a problem getting it setup right so it builds happily

**Comment by sivaprabug** [comment]
I also facing the same issue @laoshaw 

For a workaround change, the  [@vue/cli-plugin-eslint](https://www.npmjs.com/package/@vue/cli-plugin-eslint) version **4.5.12 to 5.0.8**

![image](https://github.com/openbmc/openbmc/assets/2444608/b0dd50f2-1314-42af-b607-24620e52f7aa)

**Comment by edtanous** [comment]
> I also facing the same issue @laoshaw

Can you please submit this fix to the webui-vue gerrit, rather than posting inline in the github bug?  That will allow the fix to be integrated and this bug closed.

**Comment by williamspatrick** [comment]
Seems like issue is reported (and maybe fixed already) to webui-vue?  If someone wants to backport a fix to 2.14, they're welcome to submit on Gerrit, but generally we don't actively maintain releases as a project.
